src/components/Game.js

```diff
@@ -1019,44 +951,163 @@ class Game extends React.Component{
     }
 
 
-    test = (props)=>{
-        console.log(props)
-        // return null;
+    allCase = (props) =>{
+        let id = props.id;
         let round = props.round;
-        let userIds = props.userIds;
```

Comment:
why remove `userIds`?

Reply:
looks like Jeffery use userIds in a global manner on line 44,179,246,249,256,259,858,859. So I just want to main his purpose. (looks like I should delete line 177 this.state.userIds). 
Hi @JeffreyCNL, do I need to replace userIds with any other variables e.g. GlobalPeopleIds in next version?

Reply:
The userIds is for my coding purpose since I was waiting for the input of all ids. If GlobalPeopleIds is the Ids that comes in when everybody is in the room and we can divide the team based on these ids, we can use GlobalPeopleIds for the next version.

Reply:
@JeffreyCNL  So is it okay to remove `let userIds = props.userIds;` here?